spigot/src/main/java/net/apartium/cocoabeans/spigot/lazies/ListenerAutoRegistration.java

```diff
@@ -62,6 +65,17 @@ public void register(String packageName) {
      * @param deep whether sub packages of given package should also be queried
      */
     public void register(String packageName, boolean deep) {
+        register(packageName, deep, List.of(), Set.of());
+    }
+
+    /**
+     * Auto discovers listeners in given package name
+     * @param packageName package name
+     * @param deep whether sub packages of given package should also be queried
+     * @param filler filler are object that could be provided in constructor of listeners
+     * @param ignore ignored classes
+     */
+    public void register(String packageName, boolean deep, List<Object> filler, Set<Class<?>> ignore) {
```

Comment:
instead of this it will be much easier to have a method like addInjectableObject and store it in the instance
description for filler is really bad please explain it again

Reply:
Could you give me better name and description?

Reply:
Maybe `injectableObjects`